tdesign/src/Icon/BaseIcon.tsx

```diff
@@ -35,11 +35,13 @@ export const makeIconStyle = ({
 }: IMakeIconStyleArgs) => {
   const svgDataURI = `url("${svgURI}")`;
 
-  const maskStyle = {
+  const maskStyle: SxProps<Theme> = {
     WebkitMaskImage: svgDataURI,
     maskImage: svgDataURI,
     WebkitMaskRepeat: "no-repeat",
     maskRepeat: "no-repeat",
+    WebKitMaskSize: "contain",
```

Comment:
_historical note:_ I don't remember the reason I explicitly included prefixes here, instead of letting autoprefixer do it (not even sure what's responsible for that – Next babel rules, I guess?). But IIRC there was some reason the prefixes were not applied otherwise.

Reply:
Good point, I cargo-culted the manual prefix without thinking about it too much. I removed the manual prefixes

![image](https://user-images.githubusercontent.com/889383/165576467-57d8d403-01d6-4279-bbf6-2197568426b3.png)

and some autoprefixer added those automatically


https://user-images.githubusercontent.com/889383/165576707-aebecb2e-a9d9-472f-84ce-1db8aae377a8.mp4

This may mean they are unnecessary :smile: I don't feel brave enough to remove them at this point :smile: Maybe in another PR

Reply:
Regarding autoprefixing, a little while back I got curious on this aspect, and I learned Next ships PostCSS+autoprefixer, separately while the MUI [docs](https://mui.com/material-ui/getting-started/supported-platforms/#css-prefixing) suggest "some CSS features" might require `[jss-plugin-vendor-prefixer](https://www.npmjs.com/package/jss-plugin-vendor-prefixer), this package does not appear to be in our repo.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/182515/165733506-534bb55a-2a4c-4ee4-a546-fd100017dfbc.png">

In case you were interested this appears to be where Next.js ships [`autoprefixer`](https://github.com/vercel/next.js/blob/f4fbb83d4a7b3f567c4a5f8f0a14515746a06449/packages/next/build/webpack/config/blocks/css/plugins.ts#L101) + PostCSS. Haven't dug thoroughly into it but I guess this is likely a contributor to the behavior you're observing. Maybe this helps the bravery situation for next time :)